client/layout/guided-tours/config-elements/link-quit.js

```diff
@@ -33,10 +33,10 @@ export default class LinkQuit extends Component {
 
 	render() {
 		const { children, primary, subtle, href, target } = this.props;
-		const classes = subtle ? 'guided-tours__subtle-button' : '';
+		const classes = subtle ? ' guided-tours__subtle-button' : '';
 		return (
 			<Button
-				className={ classes }
+				className={ 'guided-tours__button-link' + classes }
```

Comment:
I think there was a comment from @gwwar on how to rewrite this — that seems to be lost now ...? I liked the proposal and think it should be rewritten in that manner.